### ISSUE TITLE
Downloading movies/shows not working

### ISSUE THREAD

**Issue by daherlihy** [nixgates/plugin.video.seren#666]
I have Kodi / Seren 2.1.4 on multiple devices (my Android 11 Samsung Galaxy S10 and 3 Android TV boxes) and none of them are allowing me to download any movie or TV show through Kodi > Seren. I get prompted on the file to confirming downloading and then another prompt to advise that the download has started.

However when I go to the download directory, there are no files there, while the download manager within Seren is also empty.

**Comment by drinfernoo** [comment]
You'll need to attach a log file, preferably a debu log.

**Comment by daherlihy** [comment]
Here's the log file: https://paste.kodi.tv/atiqosimog

Let me know if you need anything else.

**Comment by drinfernoo** [comment]
To clarify, you are using RD?

**Comment by daherlihy** [comment]
Yes I am indeed. Can share a screeshot for verification if required.

**Comment by daherlihy** [comment]
Any update on this? Have I missed something or can I help any further on this?

**Comment by drinfernoo** [comment]
I haven't tested on Android yet, but this is definitely working fine on PC 🤔 

Your log doesn't show you even trying to download anything though... there should be log statements showing when the download is initiated, when the file is created on disk, and when the download completes.

I basically only see you scraping for *one* thing, not getting any playable sources, and then going to the Download Manager to see if anything happened.

Are you getting any sources? Have you tried actually using the download feature?

**Comment by daherlihy** [comment]
I do want to change this to a different location other than the default, i.e. /storage/emulated/0/Download, however I can't go any location outside of the Kodi installation folder in the Android > Data folder. So I'm suspecting this might be the source of the issue

**Comment by drinfernoo** [comment]
@daherlihy It's possible that there's some issue when the user opens that setting and then can't go anywhere from there. Also, you may have success by pressing "back" rather than "up".

**Comment by daherlihy** [comment]
I've done a video here showing my steps in trying to download a movie: https://photos.app.goo.gl/5KzcovLiwtvb7awb8

Also the back button (which as you can see throughout the video is a swipe-gesture from the right-side of the screen) doesn't allow me to go outside the Kodi Android Data folder.

**Comment by drinfernoo** [comment]
There's something strange going on in your video... namely, you do not get the dialog to choose which file to download. This should happen even with sources that only contain a single file.

**Comment by daherlihy** [comment]
I just selected the first source and then went to download on the right - not sure if this is what you're referring to (?) ... I'm using a touch screen (my phone).

If you're sure there's something else I'm missing, by any chance you could a video demo of this for me? The experience I get in my video above is the same that I get on 3 other Android TV boxes I have at home
